packages/libjit/build.sh

```diff
@@ -14,7 +14,6 @@ termux_step_extract_package() {
 		mkdir -p $TMP_CHECKOUT
 
 		git clone \
-			#--branch v$TERMUX_PKG_VERSION \
```

Comment:
How it will check out specific version ?
We do not allow stright cloning of the latest commit as will break reproducible builds.

Reply:
Also I should note that latest git is generally unwanted as can be unstable or contain incomplete work.